### DISCUSSION TITLE
Graphics API version mismatch

### DISCUSSION THREAD

**Discussion by jthomasmock** [r-universe-org/help#144] [Q&A] [answered]
Howdy! I'm new to R-Universe and was successfully able to get most of my package working as intended at [https://jthomasmock.r-universe.dev/ui#package:gtExtras](https://jthomasmock.r-universe.dev/ui#package:gtExtras).

I am getting a build error from the Vignettes as seen in [the Build source package step](https://github.com/r-universe/jthomasmock/runs/6012222508?check_suite_focus=true):

```
 --- re-building ‘plots-in-gt.Rmd’ using rmarkdown
  Quitting from lines 73-78 (plots-in-gt.Rmd) 
  Error: processing vignette 'plots-in-gt.Rmd' failed with diagnostics:
  Graphics API version mismatch
  --- failed re-building ‘plots-in-gt.Rmd’
```
I'm generating SVG plots via `svglite` + `ggplot2`, so the packages are available but I am still getting that "Graphics API version mismatch" error. 

I know the R engine was updated in 4.1 but is there something I'm missing on my side in the package that needs to change specifically for R-Universe?

**Comment by jeroen** [comment, marked as answer]
Yes this is my fault, don't worry about it. I am currently testing with R 4.2, but we still are using binary packages for 4.1 because RSPM doesn't have 4.2 yet. It will be fixed in a few days!

**Reply by jthomasmock** [reply]
Ok 😌

Thanks Jeroen and kudos on the good "get started" docs!

**Reply by jeroen** [reply]
The graphics API version stuff should be all fixed now!

**Reply by jthomasmock** [reply]
All builds are successful, thanks for note!
